server/package.json

```diff
@@ -31,9 +32,9 @@
   "devDependencies": {
     "eslint": "^8.47.0",
     "eslint-config-prettier": "^9.0.0",
+    "husky": "^8.0.3",
     "lint-staged": "^14.0.1",
-    "prettier": "^3.0.1",
-    "husky": "^8.0.3"
+    "prettier": "^3.0.1"
```

Comment:
why the change here?

Reply:
"express-rate-limit": "^7.1.1" - for the rate limit ! any other things kindly reply?

Reply:
this is what the screenshot says, your answer isn't related to the question:
![image](https://github.com/Dun-sin/Whisper/assets/78784850/0c0446c9-85a1-42c3-b180-fc60f433345b)